tests/adapters_t/sites_t/test_fakesite.py

```diff
@@ -3,7 +3,7 @@
 from tardis.interfaces.siteadapter import ResourceStatus
 from tardis.utilities.attributedict import AttributeDict
 
-from ...utilities.utilities import run_async
```

Comment:
Fixed this issue with unittest imports, which is not related to the deprecation of Python 3.6 and 3.7